### DISCUSSION TITLE
Step Down Platine

### DISCUSSION THREAD

**Discussion by Marow** [frickelzeugs/FingerprintDoorbell#2] [Q&A]
Vorweg: Sehr geiles Projekt! 
Heute im Video auf Youtube gesehen!

Hast du einen Link zu dem Step Down Converter? Daran schließt du dann an +/- den Strom an, der bei der Klingel ankommt oder?
Braucht der ESP32 dann keinen USB Anschluss?

Normalerweise hat die Klingel ja nur 2-3 Drähte, die dort ankommen.

Weitere Frage, würde das auch mit dem Board gehen?
https://www.amazon.de/gp/product/B0754N794H

Hätte ich noch rumliegen. Nur woran den WAKEUP draht und die Klingel?

**Comment by frickelzeugs** [comment]
Hi und danke! Ich hatte den hier bestellt:
https://amzn.eu/d/9aJ6q03
Leider scheint der Hersteller das Produkt überarbeitet zu haben, die neuen haben eine Spule mit dem Aufdruck "100" verbaut, ich hatte noch die mit "4R7". Die aktuellen Rezension der 100er Version lesen sich sehr durchwachsen, teilweise brennen die durch und zerstören auch das was dahinter hängt. Musst mal schauen, ob das stimmt oder ggf. noch die alte Version zu haben ist. Achtung: Die ADJ Leiterbahn muss durchtrennt werden und an der gewünschten Spannung ein Lötpunkt gesetzt werden, wenn man anstatt des Potis diese nutzen will. 

An den Step-Down Converter kommt dann in meinem Fall +24v Gleichspannung, dafür reichen 2 Drähte (Achtung: der evtl. vorhandene Klingeltrafo liefert vmtl. Wechsel- statt Gleichspannung). Der ESP8266 funktioniert nicht, es braucht einen ESP32. Wenn du die Klingel über Kabel auslösen möchtest brauchst du natürlich noch etwas an Schaltung für die Last danach (ein Relais oder Optokoppler o.Ä.) da der Ausgang des ESP32 natürlich nur 3,3v liefert und auch keine hohen Ströme. Ich bin die ersten Monate aber komplett ohne ausgekommen und habe die Klingel nur über das MQTT Signal der FingerprintDoorbell angesteuert (z.B. über Openhab, HomeAssistant etc.). Erst nachdem mein WLAN mal ausgefallen ist und die Klingel dann nicht funktionierte gab es Beschwerden und ich musste nachbessern ;-)

**Reply by henfri** [reply]
Hello,

although I am german, I write english, so that others can benefit :-)
Would it be possible to compile for ESP8266? I am asking because I have this module:
https://www.tindie.com/products/nardev/esp-rfid-relay-board-12v-for-esp8266-board/ which also brings a relay for my door-Opener...

Greetings,
Hendrik

**Reply by frickelzeugs** [reply]
Hi Hendrik,

no, the code won't compile for the ESP8266, but with some effort it should be possible to port the code to the esp8266 platform, however, I currently have no plans to do so.

I would also like to point out that it is not a good idea if the relay for controlling the door opener is accessible from (or nearby) the outside together with the fingerprint sensor. The door opening circuit should be housed separately in a safe place. Otherwise, a strong magnet could be enough to open the door ;-) See :

https://www.youtube.com/watch?v=XXW27KKHtc8

Greetings,
Tobias

**Reply by henfri** [reply]
Ok, thanks. I do not have the competence to port to the 8266 and I understand, that you have no plans.

So, you have the ESP32 behind your "Hausnummer" and the Relay somewhere else, I suppose?

I would put the ESP in the basement (~15m of wire) and the Relay right next to it. Do you think, that would work?

The plug is a bit annoying.. These Micro JST connectors are not easy to solder, I think.

**Reply by frickelzeugs** [reply]
Hi Hendrik, right after the video was published our vacation started, so my response times will be laggy till next week ;-) Yes, the relay is placed in the "Schaltschrank" in the cellar, so no attacker should be able to trigger it manually. For the Micro JST connector I ordered a female plug and glued it (not soldered it) on the board. Maybe this is also an option for you:
https://www.ebay.de/itm/184820904916?var=692589928280

**Reply by henfri** [reply]
hehe, I ordered just the same from the same seller yesterday :-)

what you mean by "glued"? I suppose you glued the casing to the board, and you made use of the fact that the wires were already soldered, right?

**Reply by frickelzeugs** [reply]
Right, only the female Plug housing was glued to the Board with superglue.

**Reply by henfri** [reply]
> Ok, thanks. I do not have the competence to port to the 8266 and I understand, that you have no plans.

I have now built the competence to port to 8266 and I got a compile (but yet untested) :-)
